lib/config/configuration.go

```diff
@@ -926,12 +922,15 @@ func Configure(clf *CommandLineFlags, cfg *service.Config) error {
 			log.Warnf("not starting the local auth service. --auth-server flag tells to connect to another auth server")
 			cfg.Auth.Enabled = false
 		}
-		addr, err := utils.ParseHostPortAddr(clf.AuthServerAddr, int(defaults.AuthListenPort))
-		if err != nil {
-			return trace.Wrap(err)
+		authServers := strings.Split(clf.AuthServerAddr, ",")
+		cfg.AuthServers = make([]utils.NetAddr, 0, len(authServers))
+		for _, as := range authServers {
+			addr, err := utils.ParseHostPortAddr(as, defaults.AuthListenPort)
+			if err != nil {
+				return trace.Errorf("cannot parse auth server address: '%v'", as)
```

Comment:
we are not using `trace.Errorf` any more, instead `trace.BadParameter` is more appropriate.

Reply:
Done.